lib/experimental/class-gutenberg-rest-block-patterns-controller.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * REST API: Gutenberg_REST_Block_Patterns_Controller class
+ *
+ * @package    Gutenberg
+ * @subpackage REST_API
+ */
+
+/**
+ * Core class used to access block patterns via the REST API.
+ *
+ * @since 6.4.0
+ *
+ * @see WP_REST_Controller
+ */
+class Gutenberg_REST_Block_Patterns_Controller extends Gutenberg_REST_Block_Patterns_Controller_6_2 {
```

Comment:
Apologies if this is obvious, but just wondering why is this extending from the `6_2` controller instead of `6_3`?

Reply:
Not obvious at all, and a great question! Looks like oversight -- I think [6.3 didn't have a block patterns controller polyfill until recently](https://github.com/WordPress/gutenberg/commits/trunk/lib/compat/wordpress-6.3/class-gutenberg-rest-block-patterns-controller-6-3.php), and I simply forgot to update. Will do now!

Reply:
bfe97c26c3